### PR TITLE
Improve client setup

### PR DESCRIPTION
**Pull request by joshuef**

### CODE REVIEW

File: nohup.out [hunk 1]
```diff
@@ -0,0 +1,142 @@
+rsync error: unexplained error (code 255) at /AppleInternal/Library/BuildRoots/a0876c02-1788-11ed-b9c4-96898e02b808/Library/Caches/com.apple.xbs/Sources/rsync/rsync/rsync.c(244) [generator=2.6.9]
```
Comment (oetyng) [inline nohup.out:1]: Should this file be included?

File: scripts/ELK/templates/metricbeat.yml [hunk 2]
```diff
@@ -64,7 +64,7 @@ setup.kibana:
   # Scheme and port can be left out and will be set to the default (http and 5601)
   # In case you specify and additional path, the scheme is required: http://localhost:5601/path
   # IPv6 addresses should always be defined as: https://[2001:db8::1]:5601
-  host: "<KIBANA-MACHINE-PUBLIC-IP>:5601" # eg: "159.65.86.95:5601"
+  host: "167.99.47.69:5601" # eg: "159.65.86.95:5601"
```
Comment (oetyng) [inline scripts/ELK/templates/metricbeat.yml:67]: Aha, I assume there is no issue hardcoding the IP like this. Are we picking one for us?

File: scripts/ELK/templates/metricbeat.yml [hunk 3]
```diff
@@ -91,7 +91,7 @@ setup.kibana:
 # ---------------------------- Elasticsearch Output ----------------------------
 output.elasticsearch:
   # Array of hosts to connect to.
-  hosts: ["<ELASTIC-MACHINE-PUBLIC-IP>:9200"] # eg: ["134.209.25.17:9200"]
+  hosts: ["188.166.96.224:9200"] # eg: ["134.209.25.17:9200"]
```
Comment (oetyng) [inline scripts/ELK/templates/metricbeat.yml:94]: This one seems we're not picking for ourselves. Will we have that same IP "always" there?

File: scripts/loop_client_tests.sh [hunk 4]
```diff
@@ -14,8 +14,8 @@ cleanup() {
 
 cd safe_network/sn_client
 
-# run 10 times, increment counter once per iter
-for i in {0..10..1}
+# run 50 times, increment counter once per iter
+for i in {0..50..1}
```
Comment (oetyng) [inline scripts/loop_client_tests.sh:18]: Oh wow, that's a nice boost in checks.
Will this run until success or actually soak it?

**Review by oetyng — APPROVED** [review]
I'm not very familiar with this part, so can't give much input, seems it's just questions to understand :) 
But lgtm!

File: scripts/loop_client_tests.sh (hunk 4 above)
Comment (joshuef) [reply scripts/loop_client_tests.sh:18]: so this just runs 50 tests regardless of outcome. We could set it much higher, but atm this has been enough to spot a mem leak

File: scripts/ELK/templates/metricbeat.yml (hunk 2 above)
Comment (joshuef) [reply scripts/ELK/templates/metricbeat.yml]: a crap. removed!

File: scripts/ELK/templates/metricbeat.yml (hunk 3 above)
Comment (joshuef) [reply scripts/ELK/templates/metricbeat.yml]: a crap. removed!

**Review by oetyng — APPROVED** [review]

File: scripts/run_client_tests_for_workspace.sh [hunk 5]
```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+# https://betterdev.blog/minimal-safe-bash-script-template/
```
Comment (jacderida) [inline scripts/run_client_tests_for_workspace.sh:3]: See just as a general rule for these scripts, I don't think we need to keep adding this boilerplate stuff. I've been writing Bash scripts for nearly ten years and never really found the need for this stuff.

Comment (joshuef) [reply scripts/run_client_tests_for_workspace.sh:3]: Okay, I've not, so not really sure. Happy to cede to your wisdom. We can do a follow up PR to clean it out in general if you like?

**Pull request merged**